Doc/library/zlib.rst

```diff
@@ -11,7 +11,7 @@ For applications that require data compression, the functions in this module
 allow compression and decompression, using the zlib library. The zlib library
 has its own home page at http://www.zlib.net.   There are known
 incompatibilities between the Python module and versions of the zlib library
-earlier than 1.1.3; 1.1.3 has a security vulnerability, so we recommend using
+earlier than 1.1.3; 1.1.3 has a `security vulnerability <http://zlib.net/zlib_faq.html#faq33>`_, so we recommend using
```

Comment:
Better to use HTTPS here:

```suggestion
earlier than 1.1.3; 1.1.3 has a `security vulnerability <https://zlib.net/zlib_faq.html#faq33>`_, so we recommend using
```

Reply:
I thought as much, but i saw a link reference somewhere earlier in that file and i thought the convention was to use `http`. I'd update this and also the other link if needed.